src/Components/Filter/Filter.jsx

```diff
@@ -0,0 +1,162 @@
+import { useFilter } from "../../Context/filterContext";
+
+export function Filter({ labels }) {
+  const { filter, filterDispatch } = useFilter();
+  console.log("label", filter.labelFilter);
+  return (
+    <button className="btn btn-secondary filter-btn" id="sort-and-filter">
+      Sort & Filter <i className="fas fa-caret-down filter-icon"></i>
+      <section className="sort-and-filter">
+        <p
+          className="clear-all"
+          onClick={(e) =>
+            filterDispatch({
+              type: "CLEAR",
+            })
+          }
+        >
+          Clear All
+        </p>
+        <label htmlFor="sort-by-date" className="filter-label">
+          Sort by Date
+        </label>
+        <div className="label-input-section">
+          <input
+            type="radio"
+            name="sort-by-date"
+            id="latest"
+            value={"latest"}
+            checked={filter.sortByDate === "latest" ? true : false}
+            onChange={(e) =>
+              filterDispatch({
+                type: "SORT",
+                payload: e.target.value,
+              })
+            }
+          />
+          <label htmlFor="latest" className="filter-label">
+            Latest Date
+          </label>
+        </div>
+        <div className="label-input-section">
+          <input
+            type="radio"
+            name="sort-by-date"
+            id="oldest"
+            value={"oldest"}
+            onChange={(e) =>
+              filterDispatch({
+                type: "SORT",
+                payload: e.target.value,
+              })
+            }
+            checked={filter.sortByDate === "oldest" ? true : false}
+          />
+          <label htmlFor="oldest" className="filter-label">
+            Oldest Date
+          </label>
+        </div>
+
+        <label htmlFor="priority" className="filter-label">
+          Priority
+        </label>
+
+        <div className="label-input-section">
+          <input
+            type="radio"
+            name="priority"
+            id="high-priority"
+            value={"High"}
+            onChange={(e) =>
+              filterDispatch({
+                type: "PRIORITY_HIGH",
+                payload: e.target.value,
+              })
+            }
+            checked={filter.priorityFilter === "High" ? true : false}
```

Comment:
Suggestion: no need to use ternary operator here, as the test condition itself will return either true or false.